Fall-2017-JS02/Challenge4.html

```diff
@@ -0,0 +1,62 @@
+<html>
+<body>
+  <script src = "https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <h2>What is your name? <label class="msg"></label></h2>
+  <div style="padding:16px;">
+    Name : <input class='name' type="textbox" value="Type Name"></input>
+  </div>
+
+  <button class="Get">Get TextBox Value</button>
+
+  <script type="text/javascript">
+  $('button').click(function () {
+    $('.msg').html($('input.name').val());
+
+    console.log("Hi");
+
+  });
+  </script>
+  <p><h2>What is your favorite food?  <label class="mss"></label></h2></p>
+
+  <div style="padding:16px;">
+    Food :<input class='food' type="textbox" value="Type Food"></input>
+  </div>
+
+  <button class="btn1">Get TextBox Value</button>
+
+  <script type="text/javascript">
+  $(".btn1").click(function () {
+    $('.mss').html($('input.food').val());
+    console.log("Hi2");
+
+  });
+  </script>
+  <p><h2>What is your favorite color?  <label class="msa"></label></h2></p>
+
+  <div style="padding:16px;">
+    Color: <input class='color' type="textbox" value="Type Color"></input>
+  </div>
+
+  <button class="btn2">Get TextBox Value</button>
+
+  <script type="text/javascript">
+  $(".btn2").click(function () {
+    $('.msa').html($('input.color').val());
+    var name = $('input.name').val();
+    var food = $('input.food').val();
+    var color = $('input.color').val();
+    console.log("Hi3");
+    $("table").append(`<tr><td>${name}</td><td>${food}</td><td>${color}</td></tr>`)
+  });
+  </script>
```

Comment:
Lines 42-51 should be in your javascript file.

Also, to help the user in using your page, you probably should just have one button - this last button is probably the best one to keep. You can move all of your other code into this function, so it would end up being something like this (in your .js file):

``` javascript
$("button.btn2").click( () => {
    // Get each of the values and store them in a variable to use later
    var name = $('input.name').val();
    var food = $('input.food').val();
    var color = $('input.color').val();

    // add each one to display next to the questions
    $('.msg').html(${name});
    $('.mss').html(${food});
    $('.msa').html(${color});

    // add the values to the table
    $("table").append(`<tr><td>${name}</td><td>${food}</td><td>${color}</td></tr>`)
});
```